lib/rouge/lexers/ghc_cmm.rb

```diff
@@ -0,0 +1,337 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+# C minus minus (Cmm) is a pun on the name C++. It's an intermediate language
+# of the Glasgow Haskell Compiler (GHC) that is very similar to C, but with
+# many features missing and some special constructs.
+#
+# Cmm is a dialect of C--. The goal of this lexer is to use what GHC produces
+# and parses (Cmm); C-- itself is not supported.
+#
+# https://gitlab.haskell.org/ghc/ghc/wikis/commentary/compiler/cmm-syntax
+#
+module Rouge
+  module Lexers
+    class GHCCmm < RegexLexer
+      title "GHC Cmm (C--)"
+      desc "GHC Cmm is the intermediate representation of the GHC Haskell compiler"
+      tag 'ghc-cmm'
+      aliases 'cmm'
```

Comment:
Should I remove this?

Reply:
Yes, I've removed it.

Reply:
Oh, sorry—I didn't explain that very well. I wondered whether it should be added. The Wikipedia page made it sound like it could be referred to as just plain C-- (which would be consistent with a `cmm` alias).

Reply:
But I guess since this is the GHC fork, that's why you don't think it's appropriate?

Reply:
IMHO the alias is fine. As far as I know, only the Haskell community actively uses Cmm. So, it should be okay to refer to the GHC dialect as "the real thing".

I've reintroduced it. Sorry, I really misunderstood you ... :smiley: